<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>FilterOpOptions</class>
 <widget class="QWidget" name="FilterOpOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>223</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item column="0" row="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KisCmbIDList" name="filtersList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSmudgeMode">
       <property name="text">
        <string>Smudge Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="1">
    <widget class="QGroupBox" name="grpFilterOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filter Settings</string>
     </property>
    </widget>
   </item>
   <item column="0" row="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>